src/safeposix/syscalls/net_calls.rs

```diff
@@ -1869,18 +2432,16 @@ impl Cage {
     ///
     /// ### Panics
     ///
-    /// * invalid or out-of-bounds file descriptor), calling unwrap() on it will
+    /// * invalid or out-of-bounds file descriptor, calling unwrap() on it will
     ///   cause a panic.
     /// * unknown errno value from socket bind sys call from libc in the case
     ///   that the socket isn't assigned an address
     /// * unknown errno value from socket listen sys call from libc
     ///
     /// for more detailed description of all the commands and return values, see
     /// [listen(2)](https://linux.die.net/man/2/listen)
-    //
-    // TODO: We are currently ignoring backlog
```

Comment:
What does this mean?

Reply:
Before we were ignoring the 'backlog' parameter of the listen_syscall, but now we are passing it to the libc call so the comment can be removed.